org-code-javabuilder/lib/src/main/java/dev/javabuilder/LocalLogHandler.java

```diff
@@ -10,22 +10,18 @@
   private final PrintStream logStream;
   private final String levelId;
   private final String channelId;
-  private final String miniAppType;
 
-  public LocalLogHandler(
-      PrintStream logStream, String levelId, String channelId, String miniAppType) {
+  public LocalLogHandler(PrintStream logStream, String levelId, String channelId) {
     this.logStream = logStream;
     this.levelId = levelId;
     this.channelId = channelId;
-    this.miniAppType = miniAppType;
```

Comment:
Unrelated to this change, I noticed that we're still logging the miniAppType, which is no longer relevant since we have separate logs for each of the mini apps.